src/main/resources/data/tconstruct/scripts/tconstruct/tinker_station.zs

```diff
@@ -0,0 +1,37 @@
+ * 5.1) You can use `/ct dump ticModifiers` to get a list of Modifiers.
+ * 5.2) The left over is in the following instance: You have a tool that has 79/80 put into the Recipe, but an Arrow gives 2, so using an Arrow would give you 81/80, so in this case it adds 1 to get 80/80 and gives you the left over so no material is lost. The normal Haste Recipe using Redstone dust and Redstone Blocks operates the same way.
+ * 5.3) For the Modifier Requirements, if there are multiple requirements (such as requiring Haste and Luck to be applied), you can use the minMatch to say how many of the requirements need to match, so setting it to 1, will mean either Luck Or Haste need to be present, both are not required.
+ * 5.4) The requirementsError is localized, so you can pass in an unlocalized string.
+ */
+
+// <recipetype:tconstruct:tinker_station>.addOverslimeModifierRecipe(name as string, ingredient as IIngredient, restoreAmount as int)
+// <recipetype:tconstruct:tinker_station>.addModifierRecipe(name as string, inputs as IIngredientWithAmount[], toolRequired as IIngredient, modifierResult as string, modifierResultLevel as int, maxLevel as int, upgradeSlots as int, abilitySlots as int)
+// <recipetype:tconstruct:tinker_station>.addModifierRecipe(name as string, inputs as IIngredientWithAmount[], toolRequired as IIngredient, modifierResult as string, modifierResultLevel as int, maxLevel as int, upgradeSlots as int, abilitySlots as int, modifierRequirements as IData, minMatch as int, requirementsError as string)
+// <recipetype:tconstruct:tinker_station>.addIncrementalModifierRecipe(String name, IIngredient input, int amountPerInput, int neededPerLevel, IIngredient toolRequirement, String modifierResult, int modifierResultLevel, int maxLevel, int upgradeSlots, int abilitySlots, IItemStack leftover)
+// <recipetype:tconstruct:tinker_station>.addIncrementalModifierRecipe(String name, IIngredient input, int amountPerInput, int neededPerLevel, IIngredient toolRequirement, String modifierResult, int modifierResultLevel, int maxLevel, int upgradeSlots, int abilitySlots, IItemStack leftover, IData modifierRequirements, int minMatch, String requirementsError)
+
+<recipetype:tconstruct:tinker_station>.addOverslimeModifierRecipe("overslime_test", <item:minecraft:dirt>, 800);
+<recipetype:tconstruct:tinker_station>.addModifierRecipe("modifier_test", [<item:minecraft:stick> * 3], <item:tconstruct:pickaxe>, "tconstruct:diamond", 2, 5, 1, 1);
+<recipetype:tconstruct:tinker_station>.addModifierRecipe("modifier_test_requirements", [<item:minecraft:apple> * 3], <tag:items:tconstruct:modifiable>, "tconstruct:emerald", 2, 5, 1, 1, {"tconstruct:haste": 1}, 1, "Haste needs to be applied first!");
```

Comment:
The requirements error is really a translation key, not sure we should encourage passing in a raw string even if it technically works.